src/chttpd/src/chttpd.erl

```diff
@@ -993,8 +1112,10 @@ error_info({conflict, _}) ->
     {409, <<"conflict">>, <<"Document update conflict.">>};
 error_info({partition_overflow, DocId}) ->
     Descr = <<
-            "Partition limit exceeded due to update on '", DocId/binary, "'"
-        >>,
+        "Partition limit exceeded due to update on '",
```

Comment:
Starting to like it. +1

Reply:
+1

Reply:
-1 Why does it sometimes end a line with the `=` sign, and other times end the line with opening symbols (`<<` in this case)? It seems inconsistent.